tests/python/pytorch/graphbolt/test_base.py

```diff
@@ -13,17 +14,20 @@
 
 @unittest.skipIf(F._default_context_str == "cpu", "CopyTo needs GPU to test")
 def test_CopyTo():
-    item_sampler = gb.ItemSampler(gb.ItemSet(torch.randn(20)), 4)
```

Comment:
Actually the original code is not a suggested way to use ItemSampler, to address the warning related to minibatch.